### ISSUE TITLE
Conformance (scope and sampling)

### ISSUE THREAD

**Issue by sydneycolemanfromgoogle** [w3c/wcag3#509]
The draft does not currently explain how to test a small sample of pages to represent a massive website. We cannot adopt this standard until a statistically valid sampling method is defined.

**Comment by hidde** [comment]
WCAG-EM 2 recommends skipping sampling in cases where the site is very small and having all pages in scope. There's also a lot of gray area in between small and massive. In my experience what sort of sample forms one that is representative becomes clear quickly when looking at a product before starting off an evaluation.

**Comment by hidde** [comment]
For WCAG 2, I would say WCAG-EM is the right place. 

For WCAG 3, we are going to be developing guidance on testing and reporting in a separate task force.

IMO, we should do this as supplemental guidance and not as part of WCAG itself, as this has worked well in WCAG 2.

Proposal: to close this issue.
